babel.config.blocks.js

```diff
@@ -0,0 +1,60 @@
+const config = {
+	presets: [
+		[
+			'@babel/env',
+			{
+				targets: { browsers: [ 'last 2 versions', 'Safari >= 10', 'iOS >= 10', 'ie >= 11' ] },
```

Comment:
Let's bring this change over:

https://github.com/Automattic/wp-calypso/pull/31464/files#diff-0e759b21864de1336af9c450cff5ad20R9

Reply:
Actually let's just leave this undefined so that Babel picks up these from `package.json`?

Reply:
Did it 👍